qa/L0_perf_analyzer_capi/json_input_data_files/float_data_with_shape.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
This duplicates files present in the L0_perf_analyzer dir

Reply:
As @CoderHam  suggests, you can reuse these files from L0_perf_analyzer. If you are worried about aesthetics then place the json_input_data_files to `qa/common` with name `perf_analyzer_test_files/json_input_data`. 
Both the solutions will be better than replicating image_data.json.